CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 - Upgrade NVIDIA Fabric Manager to version 470.141.03.
 - Disable cron job tasks man-db and mlocate, which may have a negative impact on node performance.
 - Add support for generating Slurm Configuration files for Compute Resources with Multiple Instance Types.
+- Change DynamoDB timeout from 10 minutes to 1 minute for Slurm compute nodes.
```

Comment:
From the point of view of the users the change should be something like:
`Reduce timeout from 50 to 5min in case of DynamoDB connection issues at compute node bootstrap.`

Reply:
Changed.